portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,22 +81,33 @@ private String convertToJsonUsingGson(ArrayList<String> comments){
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String text = request.getParameter("comment");
     long timeStamp = System.currentTimeMillis();
-
-    if(text.length() != 0) {
-        comments.add(text);
-    }
+    
 
     Entity taskEntity = new Entity("Task");
     taskEntity.setProperty("comment", text);
     taskEntity.setProperty("timestamp", timeStamp);
+    
 
     datastore.put(taskEntity);
     
-    response.sendRedirect("/gallery.html");
+    response.sendRedirect("/index.html");
 
     
   }
-    
+
+  private int getNumberOfComments(HttpServletRequest request){
+    String parameterValue = request.getParameter("parameterValue");
+    System.out.println(parameterValue);
+
+    int parameter = 0;
+      
+    try {
+        parameter = Integer.parseInt(parameterValue);
+    } catch (NumberFormatException e){
+        System.err.println("Can't convert to number");
```

Comment:
One tip for this is to include the value you couldn't convert in this error message. It'll make it easier to debug if that happens.